less/partials/supporters.less

```diff
@@ -76,6 +76,7 @@ body.supporters {
             font-weight: bold;
             text-transform: uppercase;
             font-family: @body-font;
+            line-height: 50px; // NOTE: equal to image height
```

Comment:
Before this change the names of supporters without images were not on the same line as supporters with images: <img width="1012" alt="screen shot 2018-08-01 at 4 09 05 pm" src="https://user-images.githubusercontent.com/171493/43546105-c08cf5a6-95a5-11e8-9160-322f4a79e248.png">